packages/components/pie-webc-core/README.md

```diff
@@ -56,3 +57,18 @@ Other dependencies may be added in future to make it easier for people to instal
 Check out our [contributing guide](https://github.com/justeattakeaway/pie/wiki/Contributing-Guide) for more information on [local development](https://github.com/justeattakeaway/pie/wiki/Contributing-Guide#local-development).
 
 To run the unit tests, simply run `yarn test --filter=pie-webc-core` from the root of the monorepo.
+
+## Testing
```

Comment:
this is so useful, thank you for adding it 🥇 

nit: I think this applies to other components and packages as well. do you think we can add this perhaps to a more global readme?

Reply:
Thanks :) I am chatting with @siggerzz and @JoshuaNg2332  about more globally applicable documentation. This is very specific to the webc core package as I've had to write some custom naming conventions to get playwright and vitest to work together nicely - I think we should figure out a global approach and then unify the documentation as you suggest 😃